src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.hpp

```diff
@@ -0,0 +1,48 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <class T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace GeneralizedHarmonic::BoundaryConditions {
+namespace detail {
+// @{
+/*!
+ * \brief Computes the expression needed to set bnudary conditions on Dt<VPsi>
+ *
+ * \details Computes the correction to the characteristic projected
+ * time-derivatives of the fundamental variables corresponding to \f$u^\psi\f$:
+ *
+ * \f{align}
+ * \partial_t u^{\psi}_{ab} = \partial_t u^{\Psi}_{ab} + v_{\psi} n^i C_{iab}
+ * \f}
+ *
+ * where \f$n^i\f$ is the local unit normal to the external boundary,
+ * and \f$C_{iab} = \partial_i \psi_{ab} - \Phi_{iab}\f$ is the
+ * three-index constraint.
```

Comment:
perhaps add an explanation that `v_{\psi}` is the characteristic speed.
It would also be helpful to indicate the difference between the capital and lowercase `\Psi` in the equation.

Reply:
I think we've been using `lambda` to denote char speeds :shrug:

Reply:
I'd favor sticking with `lambda` for char speeds, but either way, please explain what `v_{\psi}` is in the help text.

Reply:
done